lib/Sema/CSDiagnostics.cpp

```diff
@@ -423,7 +423,7 @@ bool MissingConformanceFailure::diagnoseAsError() {
   // with it and if so skip conformance error, otherwise we'd
   // produce an unrelated `<type> doesn't conform to Equatable protocol`
   // diagnostic.
-  if (isPatternMatchingOperator(const_cast<Expr *>(anchor))) {
+  if (anchor && isPatternMatchingOperator(const_cast<Expr *>(anchor))) {
```

Comment:
Could you please change it so `isPatternMatchingOperator` and `isStandardComparisonOperator` both accept `ASTNode` instead? I just overlooked that while changing other things, that's why it's still using `const_cast` instead of `castToExpr` :/

Reply:
Got it, will do!

Reply:
✅

Reply:
Thank you!